### ISSUE TITLE
Change Unauthorized message

### ISSUE THREAD

**Issue by wesleyvicthor** [FriendsOfSymfony/FOSRestBundle#1353]
Well I have tried a few approaches to change the Unauthorized message I have been receiving from the server. Basically this messages comes when a password is invalid but the customer email exists.

I need to change that message but can't find a proper way to do that, I have tried using fos_rest.exception config like this:
```YAML
exception:
        enabled: true
        codes:
            'TT\ApiFrontBundle\Exceptions\UnauthorizedException': 401
        messages:
            'TT\ApiFrontBundle\Exceptions\UnauthorizedException': true
```

I tried to change the ExceptionListener and the authentication provider but I still receive the same message, this is the structure:

```json
error: {code: 401, message: "Unauthorized", exception: [,…]}
code: 401
exception: [,…]
message: "Unauthorized"
```

How can I change this Unauthorized message ?

**Comment by lsmith77** [comment]
this feature does not make it possible to change the message .. it only makes it configurable if the message should be shown at all and what http status code should be used.

to customize the actual message you need to override the exception controller

**Comment by Ener-Getick** [comment]
Can you take a look at [this](https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1358) and tell me if this is enough to solve your issue please ?

**Comment by wesleyvicthor** [comment]
@Ener-Getick yup, this looks good, when I get back to that I will give a try. btw your point with the template engine makes sense, apparently.

**Comment by Ener-Getick** [comment]
@wesleyvicthor ok thanks, I'll wait for more feedbacks and maybe this will be included in 2.0 ☺
